src/components/quizPage/QuizPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const QuizPage: React.FunctionComponent = () => {
+    return (
+        <div>
+            <h1>This is the Quiz Component</h1>
+        </div>
+    );
+}
```

Comment:
Please add a semicolon here
```suggestion
};
```